terraform-example/README.md

```diff
@@ -0,0 +1,47 @@
+## This example and documentation are for the Beta v.1 of the Terraform Repositories, available exclusively to JFrog Beta customers.
+
+# Terraform Example
+
+## Overview
+This example demonstrates how to publish terraform modules to Artifactory.
+
+## Before Running the Example
+### Set Up the Environment
+1. Create local repository in Artifactory.
+2. Make sure version 2.12.0 or above of [JFrog CLI](https://jfrog.com/getcli/) is installed.
+
+### Validate the Setup
+In your terminal, validate that the following commands work.
+```console
+Output JFrog CLI version:
+> jfrog --version
+```
+
+### Scanning filesystem and packing modules
+File system is being scanned from the current working directory.
+A directory that contains at least one file with a “.tf” extension is considered as a terraform module and all its content will be packed in one zip file (including submodules directories) and will be deployed to Artifactory.
+There won’t be a recursive scanning inside the module directory after packing, that means subnodules won’t be packed and deploy separately to Artifactory.
+
```

Comment:
I suggest rephrasing the above section content to the below - 

The ```jf tf``` command scans the local file-system under the current working directory recursively.
It searches for directories which includes at least one file with a ```.tf``` extension. Such a directory is assumed to be a terraform module, and it is therefore packed into one zip file (including submodules directories) and then published to Artifactory.
There  isn't any recursive scanning inside the module directory after it is packed. This means that sub-nodules aren't packed and deployed separately to Artifactory.